robottelo/cli/subscription.py

```diff
@@ -35,8 +34,6 @@ class Subscription(Base):
     def upload(cls, options=None, timeout=None):
         """Upload a subscription manifest."""
         cls.command_sub = 'upload'
-        if bz_bug_is_open(1669186) and (timeout is None or timeout < 1500):
-            timeout = 1500
```

Comment:
@rochacbruno earlier this line was 
`timeouttimeout = 1500 if bz_bug_is_open(1339696) else 300 = 1500 if bz_bug_is_open(1339696) else 300` Do you think we  need  to keep it 300 ?

Reply:
Hi @omkarkhatavkar this line was

```py
if bz_bug_is_open(1669186) and (timeout is None or timeout < 1500):
    timeout = 1500
```

So the first condition was already evaluating to false `if bz_bug_is_open(1669186)` then `timeout = 1500` this was never being set since the BZ was closed 5 months ago.

Reply:
okay, make sense.